# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: taskcluster/ci/github-sync/kind.yml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/taskcluster/ci/github-sync/kind.yml b/taskcluster/ci/github-sync/kind.yml
--- a/taskcluster/ci/github-sync/kind.yml
+++ b/taskcluster/ci/github-sync/kind.yml
@@ -25,28 +25,16 @@ job-defaults:
     worker:
         max-run-time: 3600
         docker-image: {in-tree: github-sync}
     run:
         sparse-profile: github-sync
         using: run-task
 
 jobs:
-    wgpu:
-        description: Sync wgpu to github mirror
-        secret: gecko/gfx-github-sync/token
-        run:
-            command: '$GECKO_PATH/tools/github-sync/sync-to-github.sh wgpu gfx/wgpu gfx-rs/wgpu bors'
-        run-on-projects: []
-        when:
-            files-changed:
-                - 'gfx/wgpu/**'
-        treeherder:
-            symbol: GhS(wgpu)
-
     webrender:
         description: Sync webrender to github mirror
         secret: gecko/gfx-github-sync/token
         run:
             command: '$GECKO_PATH/tools/github-sync/sync-to-github.sh webrender gfx/wr servo/webrender @bors-servo'
         when:
             files-changed:
                 - 'gfx/wr/**'